lib/test/PoleEffectTest.cpp

```diff
@@ -0,0 +1,70 @@
+#include <gtest/gtest.h>
+
+#include "PoleEffect.hpp"
+
+class PoleEffectTest : public ::testing::Test {};
+
+TEST_F(PoleEffectTest, LightsHaveDifferentColors) {
+  Poles poles;
+  PoleEffect::SetPoleEffect(poles, 0);
+
+  CRGB first(poles[0].get_pole_lights()[0]);
+  CRGB second(poles[0].get_pole_lights()[1]);
+
+  ASSERT_NE(first.r, second.r);
```

Comment:
You should probably use `EXPECT_NE` here, so that we can see all three values if they're not equal.